openhtf/io/output/print_ToConsole.py

```diff
@@ -0,0 +1,89 @@
+"""Module to display test summary on console."""
+import os
+import sys
+from openhtf import util
+from openhtf.io import test_record
+from openhtf.util import measurements as mea
+import datetime
+class PrintToConsole():
+  """Print test results with failure info on console. """
+
+  def __init__(self):
+    if os.name == 'posix':    #Linux and Mac
+      self.RED =  '\033[91m'
+      self.GREEN =  '\033[92m'
+      self.ORANGE = '\033[93m'
+      self.RESET =  '\033[0m'
+      self.BOLD =  '\033[1m'
+    else: 
+      self.RED    = ""
+      self.GREEN  = ""
+      self.ORANGE = ""
+      self.RESET  = ""
+      self.BOLD   = ""
+
+    self.color_table = {
+        'PASS':self.GREEN,
+        'FAIL':self.RED,
+        'ERROR':self.ORANGE
+  }
+  # pylint: disable=invalid-name
+  def __call__(self, record):
+    indent2 = ''.join(('\n', ' '*2))
+    indent4 = ''.join(('\n', ' '*4))
+  
+    status = str(record.outcome)[-5:]
+    if status[0] == '.':
+      status = status[1:]
+
+    color = self.color_table[status]
+
+    output_lines = []
+    msg = ''.join((os.path.basename(sys.argv[0]), ': ', status))
+    msg = ''.join((color, self.BOLD, msg, self.RESET))
+    
+    output_lines.append(msg)
+
+    if status == 'FAIL' or status == 'ERROR':      
+      phases=record.phases
+      for phase in phases:
+        phase_name = phase.name
+        phase_time = str((float(phase.end_time_millis) - \
+                               float(phase.start_time_millis))/1000)
+        if status == 'FAIL':
+          measured = phase.measured_values
+          measurements = phase.measurements
+          for measurement in measurements:
+            res = measurements[measurement]
+            if res.outcome == mea.Outcome.FAIL:
+              if len(phase_name) > 0:
+                text = ''.join(('\nfailed phase: ', phase_name, ' [time taken(s): ', \
```

Comment:
`\` isn't needed to join lines that are already implicitly joined.

If there are a small number of variables being inserted into a proportionally larger amount of hard-coded text, consider using `%s`-style string formatting:
`text = 'failed phase: %s [time taken(s): %s' % (phase.name, phase_time)` etc.

I think the explicit `\n` can go away if the entries of output_lines are crafted right.